views/macros/image.html

```diff
@@ -1,16 +1,17 @@
 {% macro render(imageObj, options= {size: 'full', alt: false, description: false} ) %}
+  {% set image = apos.images.first(imageObj) %}
   <div class="o-image__wrapper{% if options.class %} {{ options.class }}{% endif %}">
-    {% if imageObj.item.attachment === undefined %}
-      <img class="o-image" src="https://via.placeholder.com/300x200" alt="{{ options.alt or imageObj.item.title }}">
+    {% if image === undefined %}
+      <img class="o-image" src="https://via.placeholder.com/300x200" alt="">
     {% else %}
-      <img class="o-image" src="{{ apos.attachments.url(imageObj.item.attachment, { size: options.size, crop: imageObj.relationship }) }}" alt="{{ options.alt or imageObj.item.title }}">
-      {% if options.description === true and imageObj.item.description %}
-          <figcaption class="o-meta o-image__caption">{{ imageObj.item.description }}</figcaption>
+      {% set url = apos.attachments.url(image, { size: options.size, crop: image._crop }) %}
```

Comment:
the crop/relationship of settings for this are not coming through, getting uncropped images like
![image](https://user-images.githubusercontent.com/1889830/62160286-934a0c00-b2e1-11e9-9b00-acdc448876ca.png)